src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,45 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { Tab } from '../types/Tab';
+import { TabsProps } from '../types/Tabs';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }: TabsProps) => {
+  const selectedTab = tabs.find((tab) => tab.id === selectedTabId) || tabs[0];
+
+  const getTabContent = () => {
+    return tabs.find((tab) => tab.id === selectedTabId)?.content;
+  };
```

Comment:
it should not be a function, just a simple const. Or you can read about and use `useMemo`